hack/scripts/render-sec-scanners-config.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+set -e
+
+# This scrpit generates the sec-scanners-config by fetching all relevant images.
+
+TAG=$1
+OUTPUT_FILE=${2:-"sec-scanners-config.yaml"}
+WEBHOOK_FILE=${3-"config/webhook/kustomization.yaml"}
+PUBLISHER_FILE=${4-"config/manager/manager.yaml"}
+
+# Fetch Webhook Image.
+echo "fetching webhook image from ${WEBHOOK_FILE}"
+WEBHOOK_IMAGE=$(yq eval '.images[0].newName' <"$WEBHOOK_FILE")
+WEBHOOK_TAG=$(yq eval '.images[0].newTag' <"$WEBHOOK_FILE")
+echo -e "webhook image is ${WEBHOOK_IMAGE}:${WEBHOOK_TAG} \n"
```

Comment:
For consistency, please create a new var `WEBHOOK_IMAGE` which consists of the `${WEBHOOK_IMAGE}:${WEBHOOK_TAG}` and use it wherever needed.

Reply:
I like the idea. Done.